scripts/commands/checkMetadata.ts

```diff
@@ -104,11 +106,8 @@ async function determineFiles(args: Arguments): Promise<[string[], string[]]> {
 }
 
 function isApi(filePath: string): boolean {
-  return (
-    filePath.includes("/api/qiskit/") ||
-    filePath.includes("/api/qiskit-ibm-runtime/") ||
-    filePath.includes("/api/qiskit-ibm-provider/")
-  );
+  const apiFolders = Pkg.VALID_NAMES.map((pkg) => `/api/${pkg}`);
```

Comment:
The new urlPaths are missing the last `/`. I think the first one will match all the rest as all start with `/api/qiskit`.
```suggestion
  const apiFolders = Pkg.VALID_NAMES.map((pkg) => `/api/${pkg}/`);
```

Reply:
Ah, good catch!